src/libConsensus/ConsensusBackup.cpp

```diff
@@ -84,15 +84,15 @@ bool ConsensusBackup::ProcessMessageAnnounce(
   // Extract and check announce message body
   // =======================================
 
-  std::vector<unsigned char> errorMsg;
+  bytes errorMsg;
   if (!m_msgContentValidator(announcement, offset, errorMsg, m_consensusID,
                              m_blockNumber, m_blockHash, m_leaderID,
                              GetCommitteeMember(m_leaderID).first,
                              m_messageToCosign)) {
     LOG_GENERAL(WARNING, "Message validation failed");
 
     if (!errorMsg.empty()) {
-      vector<unsigned char> commitFailureMsg = {
+      bytes commitFailureMsg = {
           m_classByte, m_insByte,
           static_cast<unsigned char>(ConsensusMessageType::COMMITFAILURE)};
```

Comment:
Can also change cases like this (`static_cast<unsigned char>`, `sizeof(unsigned char)`, and anywhere else where `unsigned char` is used).